### PR TITLE
DAC and Electro-pneumatic regulator functionality

### LINKED ISSUE #24: Add E-pneumatic support
Add I2C DAC drive and handle pressure input for Electropneumatic regulator.

### PR DESCRIPTION
**Pull request by riningerr**
Files added for DAC usage (MCP4725 12-bit). Allows user to output a voltage between 0 and Vdd (0 to 5V currently) from DAC. Communication over I2C using Arduino's Wire library (such shame...)

Files added for electro-pneumatic regulator that utilize the DAC code to set the output pressure of the pneumatics.

A new M-code M236 was defined in Marlin_main.cpp that allows you to set the pressure. This M-code takes a floating point pressure as an argument.

A new lookup table was created for use with the PT1200 pressure sensor. It was previously using the wrong LUT.

Fixes #24 

such pneumatics. very pressure.

breaking-doge

### CODE REVIEW

File: Marlin/Configuration.h [hunk 1]
```diff
@@ -165,11 +165,11 @@ Here are some standard links for getting your machine calibrated:
...
-#define TEMP_SENSOR_0 5
```
Comment (jminardi) [inline Marlin/Configuration.h]: Dummy value left in

File: Marlin/Configuration.h [hunk 2]
```diff
@@ -165,11 +165,11 @@ Here are some standard links for getting your machine calibrated:
...
+#define TEMP_SENSOR_0 998
 #define TEMP_SENSOR_1 0
 #define TEMP_SENSOR_2 0
 #define TEMP_SENSOR_3 0
-#define TEMP_SENSOR_BED 14
```
Comment (jminardi) [inline Marlin/Configuration.h]: Correct bed LUT should be put back

File: Marlin/Marlin_main.cpp [hunk 3]
```diff
@@ -4130,6 +4143,39 @@ inline void gcode_M226() {
   }
 #endif
  
+#ifdef DAC_I2C
+  /**
+   * M236 - Send Value to ADC w/ no EEPROM write *TESTING*
+   */
+  inline void gcode_M236() {
+    if(code_seen('S')) {
```
Comment (jminardi) [inline Marlin/Marlin_main.cpp]: If no `S` is specified I think we should echo the current value.

**Comment by jminardi** [comment]
Other than my comments this LGTM

File: Marlin/Marlin_main.cpp (hunk 3 above)
Comment (riningerr) [inline Marlin/Marlin_main.cpp]: Agreed, but as is it will work properly. I will add this functionality very soon.

**Pull request merged**